### PR TITLE
Make machine-launch fail on error. 

### PR DESCRIPTION
**Pull request by davidbiancolin**
CI will correctly pick up problems and the log will end at the problematic command. 

h/t @abejgonzalez

### CODE REVIEW

**Comment by davidbiancolin** [comment]
And because of on going weirdness with ius they've tested this for me.

File: scripts/machine-launch-script.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+set -e
```
Comment (abejgonzalez) [inline scripts/machine-launch-script.sh:3]: Would this be better as a `set -ex` so that you also get to what command runs and its args?

**Comment by abejgonzalez** [comment]
Wow. This failure where the `https://centos7.iuscommunity.org/ius-release.rpm` can't be opened is now just happening (this didn't error out 8hrs ago). Going to the link points to this issue: https://github.com/iusrepo/announce/issues/18

**Comment by abejgonzalez** [comment]
You should change the `https://centos7.iuscommunity.org/ius-release.rpm` link to `https://repo.ius.io/ius-release-el7.rpm`

**Comment by davidbiancolin** [comment]
Cool, easy enough.

**Pull request merged**